[2025-04-01T02:52:07.712+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-01T02:52:08.136+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 2-Cargar_data.decide_next_task manual__2025-04-01T02:52:02.101837+00:00 [queued]>
[2025-04-01T02:52:08.159+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 2-Cargar_data.decide_next_task manual__2025-04-01T02:52:02.101837+00:00 [queued]>
[2025-04-01T02:52:08.159+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-01T02:52:08.193+0000] {taskinstance.py:2890} INFO - Executing <Task(BranchPythonOperator): decide_next_task> on 2025-04-01 02:52:02.101837+00:00
[2025-04-01T02:52:08.226+0000] {standard_task_runner.py:72} INFO - Started process 2670 to run task
[2025-04-01T02:52:08.245+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', '2-Cargar_data', 'decide_next_task', 'manual__2025-04-01T02:52:02.101837+00:00', '--job-id', '99', '--raw', '--subdir', 'DAGS_FOLDER/Cargar_datos.py', '--cfg-path', '/tmp/tmpvtgmao9u']
[2025-04-01T02:52:08.252+0000] {standard_task_runner.py:105} INFO - Job 99: Subtask decide_next_task
[2025-04-01T02:52:08.437+0000] {task_command.py:467} INFO - Running <TaskInstance: 2-Cargar_data.decide_next_task manual__2025-04-01T02:52:02.101837+00:00 [running]> on host 7ed3aa8fcef8
[2025-04-01T02:52:08.602+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='2-Cargar_data' AIRFLOW_CTX_TASK_ID='decide_next_task' AIRFLOW_CTX_EXECUTION_DATE='2025-04-01T02:52:02.101837+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-01T02:52:02.101837+00:00'
[2025-04-01T02:52:08.604+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-01T02:52:08.635+0000] {baseoperator.py:423} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-04-01T02:52:13.660+0000] {python.py:240} INFO - Done. Returned value was: load_data
[2025-04-01T02:52:13.660+0000] {branch.py:38} INFO - Branch into load_data
[2025-04-01T02:52:13.662+0000] {skipmixin.py:233} INFO - Following branch ('load_data',)
[2025-04-01T02:52:13.691+0000] {skipmixin.py:281} INFO - Skipping tasks [('stop_task', -1)]
[2025-04-01T02:52:13.753+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-01T02:52:13.754+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=2-Cargar_data, task_id=decide_next_task, run_id=manual__2025-04-01T02:52:02.101837+00:00, execution_date=20250401T025202, start_date=20250401T025208, end_date=20250401T025213
[2025-04-01T02:52:13.878+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-01T02:52:13.926+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-01T02:52:13.929+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
